content/blog/2021-09-31-out-in-the-open-september-2021.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Out in the Open - September 2021"
+author: "Tim Fangmeyer & Knut Hühne - OK Lab Berlin, Klara Juhl - OK Lab Osnabrück"
+date: '2021-09-29T10:00:00+02:00'
+excerpt: Monatlich erscheinender Überblick über Entwicklungen in der Open Data and
+  Civic Tech Szene
+topic: community
+images:
+- imgname: out-in-the-open-2021-09/header.jpg
+  attribution: Foto von [Luke Southern](https://unsplash.com/@lukesouthernr) auf [Unsplash](https://unsplash.com/photos/4kCGEB7Kt4k)
+og_image: out-in-the-open-2021-09/header.jpg
+type: blog
+---
+Nach der Bundestagswahl im September ist der Wahlkampf vorbei, die Parteien schalten in den Verhandlungsmodus und suchen nach neuen Möglichkeiten, wie zukünftig gemeinsam regiert werden kann. Dass das Thema Digitalisierung im Wahlkampf nicht die Aufmerksamkeit erhalten hat, die es verdient, ist auch Teil unserer Blogreihe für den Monat September.
+
+In einem [Kommentar in der Süddeutschen Zeitung beschreibt der Journalist Andrian Kreye](https://www.sueddeutsche.de/meinung/digitalisierung-bundestagswahl-parteien-demokratie-internet-1.5413572), wie das Thema Digitalisierung in den letzten Wochen vor allem als Infrastrukturproblem ("flächendeckender Handyempfang") behandelt wurde. Auch wenn Kreye in der zunehmenden Digitalisierung viele Bedrohungen erkennt, hebt er gleichzeitig hervor, wie sie das Leben aller Menschen in jedem nur erdenklichen Aspekt verändern wird.
+
+Der Albtraum für jede:n Fahrradliebhaber:in – das eigene Rad wurde gestohlen. Die Zahl der Fahrraddiebstähle ist nach wie vor hoch, die Aufklärungsquote gering. Die Berliner Polizei veröffentlicht daher seit Anfang September tagesaktuelle Informationen zu Fahrraddiebstählen auf dem [Open Data Portal der Stadt](https://daten.berlin.de/datensaetze/fahrraddiebstahl-berlin). Die Veröffentlichung der Daten solle "Programmierer herausfordern, diese Daten zu nutzen und Anwendungen zu entwickeln, auf die wir noch gar nicht gekommen sind", so zitiert [Golem](https://www.golem.de/news/berlin-polizei-testet-open-data-gegen-fahrradklau-2109-159424.html) Stefan Redlich, Vizechef des Landeskriminalamtes. Erste Anwendungen, die auf den bereitgestellten Daten beruhen, ließen nicht lange auf sich warten: So entwickelte u. a. Jan Lemcke in seiner freien Zeit eine [Übersichtskarte der Rad-Diebstähle](https://fahrraddiebstahl-berlin.de/). Der dazugehörige [Twitter-Bot](https://twitter.com/DiebstahlBerlin) gibt jeden Tag Auskunft darüber, wie viele Fahrräder welchen Typs gestohlen wurden und wie hoch die maximale Schadenshöhe war.
```

Comment:
Sind für Karte und Twitter-Bot Links zum Code verfügbar? Die wären vielleicht noch schön wenn die verlinkt wären.

Reply:
Der Code wird in Zukunft noch online gestellt, ist aber leider noch nicht verfügbar.